packages/@atjson/document/src/collection.ts

```diff
@@ -47,10 +47,17 @@ export class Collection {
       this.annotations = this.annotations.sort(sortFunction);
     } else {
       this.annotations = this.annotations.sort((a, b) => {
-        if (a.start - b.start === 0) {
-          return a.end - b.end;
+        let startDelta = a.start - b.start;
+        let endDelta = a.end - b.end;
+        if (startDelta === 0) {
+          if (endDelta === 0) {
+            return a.type.localeCompare(b.type);
```

Comment:
`localeCompare` will change according to the user's locale (and the machine's locale). Could we instead use `<` / `>` operators to return `1` / `-1` / `0`?

Reply:
working on it now